tests/sweep_framework/runner.py

```diff
@@ -27,6 +27,20 @@ def git_hash():
         return "Couldn't get git hash!"
 
 
+def get_hostname():
+    try:
+        return subprocess.check_output(["uname", "-n"]).decode("ascii").strip()
+    except:
+        return "Couldn't get hostname!"
+
+
+def get_username():
+    try:
+        return os.environ["USER"]
+    except:
+        return "Couldn't get username!"
```

Comment:
I wouldn't try to recover from these exceptions. If you can't get the hostname or username, there must be some critical failure.